spec/features/merchant_dashboards/index_spec.rb

```diff
@@ -0,0 +1,115 @@
+      @invoice_3 = @customer_3.invoices.create(status: 'Completed')
+      @invoice_4 = @customer_4.invoices.create(status: 'Completed')
+      @invoice_5 = @customer_5.invoices.create(status: 'Completed')
+      @invoice_6 = @customer_6.invoices.create(status: 'Completed')
+      @invoice_7 = @customer_6.invoices.create(status: 'Completed')
+
+
+      @invoice_item_1 = @invoice_1.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'shipped')
+      @invoice_item_2 = @invoice_2.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'shipped')
+      @invoice_item_3 = @invoice_3.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'shipped')
+      @invoice_item_4 = @invoice_4.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'shipped')
+      @invoice_item_5 = @invoice_5.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'shipped')
+      @invoice_item_6 = @invoice_6.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'packaged')
+      @invoice_item_7 = @invoice_7.invoice_items.create(item_id: @item.id, quantity: 1, unit_price: 1000, status: 'pending')
+
+
+
+      @transaction_1 = @invoice_1.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_2 = @invoice_1.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_3 = @invoice_2.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_4 = @invoice_2.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_5 = @invoice_3.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_6 = @invoice_4.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_7 = @invoice_4.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_8 = @invoice_5.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_9 = @invoice_5.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_10 = @invoice_5.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'success')
+      @transaction_11 = @invoice_6.transactions.create(credit_card_number: 1234123412341234, credit_card_expiration_date: '2012-03-27', result: 'failed')
+
+      visit "/merchants/#{@merchant.id}/dashboard"
```

Comment:
Do we want to use merchant_dashboard_path for this?

Reply:
yeah i think that's a good call. we don't have resourceful routing setup yet but will add it in soon